lib/index.js

```diff
@@ -58,7 +58,8 @@ var epilogue = {
       pagination: args.pagination,
       updateMethod: this.updateMethod,
       search: args.search,
-      sort: args.sort
+      sort: args.sort,
+      reloadOnCreate: args.reloadOnCreate
```

Comment:
@mbroadst What we do ? 
This by default ? Or nothing ^^
```
reload: {
  onCreate: false,
  onUpdate: true
}
```
But on update,  it is a reload only if needed